app/controllers/api/api_controller.rb

```diff
@@ -70,16 +75,31 @@ def server_info
       }
     end
 
-    def product_info
+    def product_info_data
       {
         :name                 => Vmdb::Appliance.PRODUCT_NAME,
         :name_full            => I18n.t("product.name_full"),
         :copyright            => I18n.t("product.copyright"),
         :support_website      => I18n.t("product.support_website"),
         :support_website_text => I18n.t("product.support_website_text"),
+        :branding_info        => branding_info
       }
     end
 
+    def branding_image_path(custom, default)
+      custom || ActionController::Base.helpers.image_path(default)
+    rescue Sprockets::FileNotFound # UI isn't loaded, we don't want images
+      nil
+    end
+
+    def branding_info
+      {
+        :brand      => branding_image_path(Settings.server.custom_brand, 'layout/brand.svg'),
+        :logo       => branding_image_path(Settings.server.custom_logo, 'layout/login-screen-logo.png'),
+        :login_logo => Settings.server.custom_login_logo ? Settings.server.custom_login_logo : nil
```

Comment:
Do we need the ternary operator here?

Reply:
Yes because it can also be `false` and that's not caught by `compact`